pkg/apiserver/utils/bcode/016_config.go

```diff
@@ -37,4 +37,7 @@ var (
 
 	// ErrNotFoundDistribution means the distribution is not exist
 	ErrNotFoundDistribution = NewBcode(404, 16007, "the distribution is not exist")
+
+	// ErrChangeAuthSwitch the template of the config can not be change
+	ErrChangeAuthSwitch = NewBcode(400, 16008, "the auth switch of the image registry config can not be change")
```

Comment:
Not only the image registry config template could change the secret type. So, this error type and description should have more universality.

Reply:
@barnettZQG ok, have done, please review